recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "5.5" %}
+{% set version = "6.1.1" %}
```

Comment:
The latest version is `6.2`

Reply:
Updated to the latest version `6.2` :

https://github.com/AnacondaRecipes/coverage-feedstock/blob/update_to_6.1.1/recipe/meta.yaml#L1